fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadTask.java

```diff
@@ -75,6 +75,7 @@ protected void exec() {
             isFinished = true;
         } catch (UserException e) {
```

Comment:
https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadLoadingTask.java#L264
this timeout throws `LoadException`, and `LoadException` extends `UserException`.

Reply:
done